### ISSUE TITLE
Dedicated multiplayer

### ISSUE THREAD

**Issue by loom-framework** [Dankan37/Arma-3-Iron-Dome#2]
It shoot down artillery rounds on multiplayer dedicated server but not possible to interract with turret. So that mean not possible to protect blufor side outgoing shells by turning off turrret.   (Tested on CRAM mod)

Is this fixed on Iron dome mod? Or will be fixed? 

Any comments?

**Comment by wersal454** [comment]
@Dankan37 maybe something like this?

```private _friendlyentities = [];
private _side = side _unit;
addMissionEventHandler ["ArtilleryShellFired", {
	params ["_vehicle", "_weapon", "_ammo", "_gunner", "_instigator", "_artilleryTarget", "_targetPosition", "_shell"];
	if (side _gunner == _side) then {
		_friendlyentities = pushBack _shell;
	};
}];
 ```

And then remove friendly entities/projectiles from the target list

**Comment by loom-framework** [comment]
That's actually fine that turret work for all. Kind of realistic.   Just need option to turn off.

**Comment by wersal454** [comment]
> That's actually fine that turret work for all. Kind of realistic. Just need option to turn off.

Yeah
I came up with this before patch 2.18 dropped and  fix for shooting down outgoing projectiles was released. Both Dankans fix and mine don't work,  Dankan as an original author should know more why it doesn't work

Also @Dankan37 add a check for an UAV with no AI in it, turrets tend to shoot at them even when they are grounded

**Comment by loom-framework** [comment]
Yes latest update broke some mods.   Have to fix to enjoy gameplay.

**Comment by Dankan37** [comment]
I will work on the fix for the turrets, I missed that!
Also yes, your idea for the side might work, I will look into it.
